modules/claims_api/lib/bgs_service/power_of_attorney_request_service.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class PowerOfAttorneyRequestService < ClaimsApi::LocalBGS
```

Comment:
can we name these after the beans, similar to the `BenefitsClaimService` and `ClaimManagementService`? We could consolidate the 8 vnp calls in to a `VNPProcService` if we're all open to that idea, but I'd honestly prefer them unique to their Bean / BGS Service.

Reply:
Can do